Extensions/quick_tags.js

```diff
@@ -67,7 +67,9 @@ XKit.extensions.quick_tags = new Object({
 
 	run: function() {
 		this.running = true;
-		if ($(".post.post_full").length <= 0) {return; }
```

Comment:
this is really huge. Can we do `.post.post_full, .post.post_brick` or something?